### PR TITLE
[ticket/12649] Change sort & display options in footers to dropdown menu

### PR DESCRIPTION
**Pull request by RMcGirr83**
PHPBB3-12649

### CODE REVIEW

File: phpBB/styles/prosilver/theme/common.css [hunk 1]
```diff
@@ -391,10 +391,6 @@ ul.linklist.bulletin > li.no-bulletin:before {
 	top: auto;
 }
 
-.dropdown-container.topic-tools {
-	float: left;
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/common.css]: why?

File: phpBB/styles/prosilver/theme/buttons.css [hunk 2]
```diff
@@ -53,15 +53,53 @@
 	border-radius: 0 4px 4px 0;
 }
 
-.search-header .button-search,
-.search-header .button-search-end {
-	border-top-width: 0;
-	border-bottom-width: 0;
-	padding: 3px 5px;
+.tools-icon:before, .modtools-icon:before, .search-icon:before, .search-adv-icon:before, .sort-icon:before {
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/buttons.css]: all changes to this file need removed use fa icon to accomplish the icons

File: phpBB/styles/prosilver/theme/bidi.css [hunk 3]
```diff
@@ -88,10 +88,6 @@
 
 /* Dropdown menu
 ---------------------------------------- */
-.rtl .dropdown-container.topic-tools, .rtl .dropdown-container-left {
-	float: right;
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/bidi.css]: why?

File: phpBB/styles/prosilver/template/display_options.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<div class="dropdown-container dropdown-container-{S_CONTENT_FLOW_BEGIN} dropdown-button-control sort-tools">
+	<span title="{L_SORT_OPTIONS}" class="button icon-button sort-icon dropdown-trigger dropdown-select"></span>
```
Comment (hanakin) [inline phpBB/styles/prosilver/template/display_options.html]: should be
````
       <span title="{L_SORT_OPTIONS}" class="button button-secondary dropdown-trigger dropdown-select">
			<i class="icon fa-sort-amount-asc fa-fw" aria-hidden="true"></i>
			<span class="caret"><i class="icon fa-sort-down fa-fw" aria-hidden="true"></i></span>
		</span>
````

**Comment by RMcGirr83** [comment]
Was having one hell of time doing the rebase.  Note to self keep fork up to date.  Will make the changes suggested and recommit.  Mea Culpa

File: phpBB/styles/prosilver/template/viewforum_body.html [hunk 5]
```diff
@@ -278,6 +264,24 @@ <h2 class="forum-title"><!-- EVENT viewforum_forum_name_prepend --><a href="{U_V
...
+			<!-- INCLUDE display_options.html -->
+			</form>
+		<!-- ENDIF -->
+
+		<!-- IF S_SELECT_SORT_DAYS and not S_IS_BOT -->
+			<form method="post" action="{S_FORUM_ACTION}">
+			<!-- INCLUDE display_options.html -->
+			</form>
+		<!-- ENDIF -->
+
```
Comment (hanakin) [inline phpBB/styles/prosilver/template/viewforum_body.html]: why is this here three times?

Comment (RMcGirr83) [inline phpBB/styles/prosilver/template/viewforum_body.html]: :scream: How in the bloody hell did that happen? :angry:

Comment (marc1706) [inline phpBB/styles/prosilver/template/viewforum_body.html]: Hm, displayed fine so not sure how that happened.

@RMcGirr83 could you please create a PR that fixes this?

Comment (RMcGirr83) [inline phpBB/styles/prosilver/template/viewforum_body.html]: Yep

**Comment by RMcGirr83** [comment]
https://github.com/phpbb/phpbb/pull/4043

**Comment by hanakin** [comment]
for reference this is related to https://github.com/phpbb/phpbb/pull/2541
